tests/bud.bats

```diff
@@ -4197,8 +4200,7 @@ _EOF
 
 @test "bud pull false no local image" {
   target=pull
-  run_buildah build $WITH_POLICY_JSON -t ${target} --pull=false $BUDFILES/pull
-  expect_output --substring "COMMIT pull"
+  run_buildah 125 build $WITH_POLICY_JSON -t ${target} --pull=false $BUDFILES/pull
```

Comment:
Should this be checking for a more specific error message, as the previous chunk does?

Reply:
Added